package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "webrisk",
```

Comment:
Please make sure `package-lock.json` is added to the `.gitignore`, and delete it from the PR

Reply:
I thought we were starting to experiment with adding `package-lock.json` again?